src/models/generated/BatteryEMS.jl

```diff
@@ -73,9 +75,11 @@ mutable struct BatteryEMS <: Storage
     reactive_power_limits::Union{Nothing, MinMax}
     "Base power of the unit in MVA"
     base_power::Float64
-    operation_cost::Union{Nothing, OperationalCost}
+    operation_cost::StorageManagementCost
     "Storage target at the end of simulation as ratio of storage capacity."
     storage_target::Float64
+    "Storage Maximum number of cycles per year"
+    cycle_limits::Int
```

Comment:
rather than having it as static value  to limit the cycles, I think it would be great if we can also define the cost for exceeding the cycling limit

Reply:
that is defined in the cost function. This is just the limit data